source/Playnite/Localization/LocSource.xaml

```diff
@@ -402,6 +413,8 @@ If disabled, background artwork is downloaded only when first needed, using less
     <sys:String x:Key="LOCSettingsShowMediaSizeWarning">Display warning when assigning too large game media</sys:String>
     <sys:String x:Key="LOCOpenDirectoryCommand">Folder open command</sys:String>
     <sys:String x:Key="LOCSettingsPreferredAgeRatingOrg">Preferred age rating organization</sys:String>
+    <sys:String x:Key="LOCSettingsScanLibInstallSizeOnLibUpdate">Scan modified install size of games on library update</sys:String>

```

Comment:
I think something like "Update install size on library update" would fit more, at least it doesn't better to me, but I'm not native speaker so...

Reply:
Done in https://github.com/JosefNemec/Playnite/pull/3017/commits/ee21d0c395bf95be8721ba98286ea8dac44576e9